client/landing/stepper/hooks/use-resend-email-verification.ts

```diff
@@ -0,0 +1,32 @@
+import { useTranslate } from 'i18n-calypso';
+import { useDispatch } from 'react-redux';
+import { successNotice, errorNotice } from 'calypso/state/notices/actions';
+import { useSendEmailVerification } from './use-send-email-verification';
+
+export function useResendEmailVerification() {
```

Comment:
@Addison-Stavlo I've moved the resend email logic into a reusable hook that we can use in both places

Reply:
Just wanted to confirm if this is the right place for this hook? One might think that hooks in client/landing/stepper/hook  are stepper specific. As this one is used more broadly, I wonder if we should relocate it to some more widely shared location. I'm not sure where that would be though. Maybe client/lib?

Reply:
@edanzer That's a great point tbh I am still unsure of what's the best place to put this. The reason I decided to put it under `/stepper/hooks` is because this is where the existing `use-send-email-verification.ts` lives. Since the two hooks are very closely related I thought placing it there makes the most sense. I am absolutely open to changing that, it's just that I don't  know the architecture well enough to suggest a better place